wallet/macos/Flutter/ephemeral/flutter_export_environment.sh

```diff
@@ -3,12 +3,10 @@
 export "FLUTTER_ROOT=/Users/jawad/Downloads/flutter"
```

Comment:
i understand this has been an issue for a while and hasn't really been looked at, but these ephemeral files shouldn't really be in source control, should they?

Reply:
i am not sure about that.